cmake/FindLAPACKE.cmake

```diff
@@ -7,185 +7,194 @@
 #
 # ::
 #
-#   find_package(LAPACKE REQUIRED)
-#   include_directories(${LAPACKE_INCLUDE_DIRS})
-#   add_executable(foo foo.cc)
-#   target_link_libraries(foo ${LAPACKE_LIBRARIES})
+# find_package(LAPACKE REQUIRED) include_directories(${LAPACKE_INCLUDE_DIRS}) add_executable(foo
+# foo.cc) target_link_libraries(foo ${LAPACKE_LIBRARIES})
 #
 # This module sets the following variables:
 #
 # ::
 #
-#   LAPACKE_FOUND - set to true if the library is found
-#   LAPACKE_INCLUDE_DIRS - list of required include directories
-#   LAPACKE_LIBRARIES - list of libraries to be linked
-#   LAPACKE_VERSION_MAJOR - major version number
-#   LAPACKE_VERSION_MINOR - minor version number
-#   LAPACKE_VERSION_PATCH - patch version number
-#   LAPACKE_VERSION_STRING - version number as a string (ex: "0.2.18")
+# LAPACKE_FOUND - set to true if the library is found LAPACKE_INCLUDE_DIRS - list of required
+# include directories LAPACKE_LIBRARIES - list of libraries to be linked LAPACKE_VERSION_MAJOR -
+# major version number LAPACKE_VERSION_MINOR - minor version number LAPACKE_VERSION_PATCH - patch
+# version number LAPACKE_VERSION_STRING - version number as a string (ex: "0.2.18")
```

Comment:
I don't love how this system reflows comments.   There are a lot of cases like this were the original spacing was much nicer to read.

Reply:
This was actually just missing `*` to be a bullet point list, see https://github.com/kotekan/kotekan/blob/4ed7042c08d31588d51028301f6ab0dbe45ed8f5/tools/cmake_format_config.py#L107